astropy/cosmology/parameter.py

```diff
@@ -75,6 +83,10 @@ def __init__(self, *, derived=False, unit=None, equivalencies=[],
                             f"{self._registry_validators.keys()}")
         self._fvalidate = fvalidate
 
+        # display stuff
+        self._fmt = str(fmt)  # @property is `format_spec`
+        self._format = format or {}  # see DisplayFormatAttributeMixin
```

Comment:
Should `DisplayFormatAttributeMixin` do that conversion of `None` to `{}`?

Reply:
As in have ``DisplayFormatAttributeMixin.__init__`` ?